apache-couchbase-mixin/dashboards/couchbase-node-overview.libsonnet

```diff
@@ -0,0 +1,1141 @@
+      placement: 'bottom',
+      showLegend: true,
+    },
+    tooltip: {
+      mode: 'multi',
+      sort: 'desc',
+    },
+  },
+};
+
+local averageScanLatencyPanel = {
+  datasource: promDatasource,
+  targets: [
+    prometheus.target(
+      'sum by(couchbase_cluster, index, instance, job) (index_avg_scan_latency{job=~"$job", couchbase_cluster=~"$couchbase_cluster", instance=~"$instance"})',
+      datasource=promDatasource,
+      legendFormat='{{couchbase_cluster}} - {{instance}} - {{index}}',
+    ),
+  ],
+  type: 'timeseries',
+  title: 'Average scan latency',
+  description: 'Average time to serve a scan request per index.',
+  fieldConfig: {
+    defaults: {
+      color: {
+        mode: 'palette-classic',
+      },
+      custom: {
+        axisCenteredZero: false,
+        axisColorMode: 'text',
+        axisLabel: '',
+        axisPlacement: 'auto',
+        barAlignment: 0,
+        drawStyle: 'line',
+        fillOpacity: 0,
+        gradientMode: 'none',
+        hideFrom: {
+          legend: false,
+          tooltip: false,
+          viz: false,
+        },
+        lineInterpolation: 'linear',
+        lineWidth: 1,
+        pointSize: 5,
+        scaleDistribution: {
+          type: 'linear',
+        },
+        showPoints: 'auto',
+        spanNulls: false,
+        stacking: {
+          group: 'A',
+          mode: 'none',
+        },
+        thresholdsStyle: {
+          mode: 'off',
+        },
+      },
+      mappings: [],
+      thresholds: {
+        mode: 'absolute',
+        steps: [
+          {
+            color: 'green',
+          },
+          {
+            color: 'red',
+            value: 80,
+          },
+        ],
+      },
+      unit: 'ns',
+    },
+    overrides: [],
+  },
+  options: {
+    legend: {
+      calcs: [],
+      displayMode: 'list',
+      placement: 'bottom',
+      showLegend: true,
+    },
+    tooltip: {
+      mode: 'multi',
+      sort: 'desc',
+    },
+  },
+};
+
+local errorLogsPanel = {
+  datasource: lokiDatasource,
+  targets: [
+    {
+      datasource: lokiDatasource,
+      editorMode: 'code',
+      expr: '{filename=~"/opt/couchbase/var/lib/couchbase/logs/error\\.log", job=~"$job", couchbase_cluster=~"$couchbase_cluster", instance=~"$instance"} |= ``',
+      queryType: 'range',
+      refId: 'A',
+    },
+  ],
+  type: 'logs',
+  title: 'Error logs',
+  description: 'Recent error logs from a node.',
+  options: {
+    dedupStrategy: 'none',
+    enableLogDetails: true,
+    prettifyLogMessage: true,
+    showCommonLabels: false,
+    showLabels: false,
+    showTime: false,
+    sortOrder: 'Descending',
+    wrapLogMessage: false,
+  },
+};
+
+local couchbaseLogsPanel = {
+  datasource: lokiDatasource,
+  targets: [
+    {
+      datasource: lokiDatasource,
+      editorMode: 'code',
+      expr: '{job=~"$job", couchbase_cluster=~"$couchbase_cluster", instance=~"$instance", filename=~"/opt/couchbase/var/lib/couchbase/logs/couchdb\\.log"} |= ``',
```

Comment:
On the Node Overview dashboard, for both log panels, would remove this `\` before the `.`
I don't think we need to escape the `.` here and when I re-import the json to test, those panels have an issue with it.

Reply:
There's still one more `\` that I would get rid of